packages/minimongo/cursor.js

```diff
@@ -216,6 +216,16 @@ export default class Cursor {
     return LocalCollection._observeFromObserveChanges(this, options);
   }
 
+  /**
+   * @summary observe async version
+   * @locus Anywhere
+   * @memberOf Promise<Mongo.Cursor>
+   * @instance
+   */
+  observeAsync(options) {
+    return new Promise(resolve => resolve(this.observe(options)));
```

Comment:
It's fine, though `Promise.resolve(this.observe(options))` would be simpler. (I assume `observe` cannot throw; if it can, then it has to stay like this.) Alternatively, this function could be `async`.

Reply:
I believe using `new Promise` instead of `Promise.resolve` in the `observe` or `observeChanges` could automatically catch any potential errors, which I don't know exactly if currently they are. Anyway, the `new Promise` pattern seems more adaptable and better for being ready to handle future logic adjustments that may throw errors.

Regarding `async`, I initially included it, but later realized it's necessary only when using `await`. When simply returning a `Promise`, it's not mandatory. This seems more like a code convenience, as other sections of the codebase omit the mention of async when it's not necessary I just removed it from this code.